src/components/common/Dialogs/index.ts

```diff
@@ -4,20 +4,20 @@ export { default as ManageExpenditureDialog } from './ManageExpenditureDialog';
 // export { default as EditDomainDialog } from './EditDomainDialog';
 export { default as CreatePaymentDialog } from './CreatePaymentDialog';
 // export { default as ManageDomainsDialog } from './ManageDomainsDialog';
-// export { default as ManageFundsDialog } from './ManageFundsDialog';
-// export { default as UnlockTokenDialog } from './UnlockTokenDialog';
-// export { default as TransferFundsDialog } from './TransferFundsDialog';
+export { default as ManageFundsDialog } from './ManageFundsDialog';
+export { default as UnlockTokenDialog } from './UnlockTokenDialog';
+export { default as TransferFundsDialog } from './TransferFundsDialog';
 // export { default as AdvancedDialog } from './AdvancedDialog';
 // export { default as PermissionManagementDialog } from './PermissionManagementDialog';
 // export { default as RecoveryModeDialog } from './RecoveryModeDialog';
```

Comment:
Recovery mode won't be available for the initial launch of the CDapp, so the imports for it might as well be removed